worker/lib/workers/qemu.ts

```diff
@@ -276,7 +285,7 @@ class QemuWorker extends EventEmitter implements Leviathan.Worker {
     const dnsmasqArgs = [
       `--interface=${bridgeName}`,
       `--dhcp-range=${this.qemuOptions.network.dhcpRange}`,
-      '--conf-file',
```

Comment:
@jakogut I'm still experimenting with this, but was there a purpose to this flag with no file?

Reply:
Disables loading the default configuration file.